### PR TITLE
 #295 : Coding standard class naming convention

### PR DESCRIPTION
**Pull request by nicolas-bastien**
@mickaelandrieu I'm on it still have some work to review should be done this we

### CODE REVIEW

File: ClassContent/AbstractContent.php [hunk 1]
```diff
@@ -435,7 +435,7 @@ public function setLabel($label)
     /**
      * Set the acceptable classname
      * @param  array                          $accept
-     * @return \BackBee\ClassContent\AContent The current instance
```
Comment (nicolas-bastien) [inline ClassContent/AbstractContent.php]: check pb AContent should be AbstractContent

**Comment by mickaelandrieu** [comment]
@nicolas-bastien it's ok :+1: , ping me when the tests are green ;)

**Comment by nicolas-bastien** [comment]
@mickaelandrieu all green now

-> hhvm nightly fails but it's already the case on master

by the way as I spent some time on your code I open a few QA issues to help you improve it.

I tried to split issues so it will be easier to fix and to revue.

**Comment by mickaelandrieu** [comment]
@nicolas-bastien thank for this work, I will review this one tomorrow :)

**Comment by nicolas-bastien** [comment]
@mickaelandrieu done

I didn't touch the phpdoc format and open another issue to try to simplify code revue and minimise conflict.

I saw you did another big PR, please merge mine first cause I won't have the time this days to fix possible conflict.

If I can suggest something you should this kind of PR one a time if you don't want to have to much conflict.

Travis is red due to github connection pb.
Test are green on my computer

**Comment by mickaelandrieu** [comment]
I relaunch travis build.

I will merge your PR before mine, don't worry about it ;)

**Comment by mickaelandrieu** [comment]
Thank you @nicolas-bastien !

**Pull request merged**